### ISSUE TITLE
predict API KeyError

### ISSUE THREAD

**Issue by patrikwm** [AI-Riksarkivet/htrflow_app#2]
Hi,

Trying out the API, and I can't get it to work. 

I'm sharing the images with `python3 -m http.server 8000`

Using `gradio_client==0.6.1`

```python
from gradio_client import Client 
client = Client("http://10.10.10.10:7860/")
job = client.submit("http://10.10.10.11:8000/my_image.jpg",api_name="/predict",)
while not job.done():
    time.sleep(0.1)
print(job.status())
print(job.exception)
try:
    print(job.result())
except KeyError as e:
    print(f"KeyError: {e}")
```

The server logs show that pipeline is done.

```bash
htrflow-htrflow-1  | Function 'transcribe' executed in 0.2464s 
2it [00:08,  4.16s/it]
htrflow-htrflow-1  | Function 'running_htr_pipeline' executed in 10.3494s
```

gradio_client shows KeyError: 'log'

```bash 
StatusUpdate(code=<Status.FINISHED: 'FINISHED'>, rank=0, queue_size=None, eta=None, success=False, time=datetime.datetime(2023, 10, 31, 19, 36, 43, 703883), progress_data=None)
<bound method Future.exception of <Job at 0x106d49a50 state=finished raised KeyError>>
KeyError: 'log'
```

**Comment by Borg93** [comment]
Hi, I updated some of the code. Can you make a git pull and see if the issues still there?

**Comment by patrikwm** [comment]
I tried with the latest commit and there were some issues.

*1. API name change* api_name="/run_htr_pipeline",

*2. Docker cuda error*
1. I modified the utils.py so I use my own hugginface repo.
2. I build the docker image.
3. I get error `RuntimeError: nms_impl: implementation for device cuda:0 not found.`


*3. Running python app.py (outside docker) is not downloading models*
1. `python3 -m venv .venv`
2. `source .venv/bin/activate`
3. `pip install -r requirements.txt`
4. `python app.py`
5.  `FileNotFoundError: ./models/RmtDet_lines/epoch_12.pth can not be found.`

The docker image is downloading the models so I copied them to the correct directory and after that the api calls to the new endpoint worked with gradio_client.
